src/components/CandidateTaskView/CandidateTaskView.js

```diff
@@ -0,0 +1,45 @@
+import { observer } from "mobx-react";
+import React from "react";
+import { Block, Elem } from "../../utils/bem";
+import "./CandidateTaskView.styl";
+
+export const CandidateTaskView = observer(({ item, columns }) => {
+  const { data } = item;
+
+  return (
+    <Block name="candidate-task-view">
+      <Elem name="data-display-container">
+        {Object.keys(data).map( (dataKey) => {
+          const columnDefinition = columns.find(colData => colData.alias === dataKey);
+          const dataValue = data[dataKey];
+          let elementDisplay;
+          
+          if (columnDefinition) {
+            if (columnDefinition.currentType === "Image") {
+              elementDisplay = (
+                <Elem name='data-display' mod={{ image: true }}>
+                  <img src={dataValue} />
+                </Elem>
+              );
+            } else {
+              elementDisplay = (
+                <Elem name='data-display' mod={{ text: true }} >
+                  {dataValue}
+                </Elem>
+              );
+            }
+          }
```

Comment:
i would also simplify this condition:

```suggestion
          if (columnDefinition?.currentType === 'Image') {
            return (
              <Elem name='data-display' mod={{ image: true }}>
                <img src={dataValue} />
              </Elem>
            );
          }
            
          return (
            <Elem name='data-display' mod={{ text: true }} >
              {dataValue}
            </Elem>
          );
```